apps/email/js/html_cache_restore.js

```diff
@@ -1,6 +1,6 @@
+'use strict';
 /*jshint browser: true */
 /*global performance, console, Notification */
-'use strict';
```

Comment:
This seems like an unnecessary change, jshint passes for me without this change, Might be good to remove it to just keep the diff smaller.